components/automate-cli/cmd/chef-automate/applications.go

```diff
@@ -168,7 +171,14 @@ func newApplicationsRemoveSvcsCmd() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "remove-svcs",
 		Short: "Remove services from the applications database",
-		RunE:  runApplicationsRemoveSvcsCmd,
+		Long: `
+Remove services from the applications database.
+
+You must decommission the services prior to running this command. Services
```

Comment:
What does "decommission" mean in this context? `hab svc stop` or `hab svc unload`? Something else? Wat specific actions does the user need to take?

Reply:
The user needs to make the service not run any more. How they do that could be retiring physical hardware, terminating a VM or container, or a `hab svc unload`.

The idea here is that the user is decommissioning some habitat services (for example, shutting down a test environment) and they wish for the services to no longer appear in the automate UI and they do not want to wait the normal amount of time before Automate removes the services automatically (default is 7 days).